<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Individual</class>
 <widget class="QDialog" name="Individual">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>781</width>
     <height>461</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="salveaza">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>550</y>
     <width>190</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Salveaza si inchide</string>
   </property>
  </widget>
  <widget class="QLabel" name="label1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>480</y>
     <width>771</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Selectati din lista de mai sus angajatul caruia doriti sa ii atribuiti obiective.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>510</y>
     <width>771</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Daca acesta are deja obiectivele definite, prin selectare se pot vizualiza si modifica.</string>
   </property>
  </widget>
  <widget class="QLabel" name="mob1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Primul obiectiv:</string>
   </property>
  </widget>
  <widget class="QLabel" name="sh">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>771</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cele trei obiective ale acestui angajat sunt:</string>
   </property>
  </widget>
  <widget class="QLabel" name="ob2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>771</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Al doilea obiectiv nu este stabilit</string>
   </property>
  </widget>
  <widget class="QLabel" name="ob1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>771</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Primul obiectiv nu este stabilit</string>
   </property>
  </widget>
  <widget class="QLabel" name="ob3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>771</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Al treilea obiectiv nu este stabilit</string>
   </property>
  </widget>
  <widget class="QLabel" name="add">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>771</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Adaugarea/modificarea obiectivelor angajatului:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ob1Edit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>721</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="mob2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Al doilea obiectiv:</string>
   </property>
  </widget>
  <widget class="QLabel" name="mob3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Al treilea obiectiv:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ob2Edit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>721</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="ob3Edit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>721</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="asob1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Obiectiv operational asociat:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="asob1Box">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>230</y>
     <width>501</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="asob2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Obiectiv operational asociat:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="asob2Box">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>330</y>
     <width>501</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="asob3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Obiectiv operational asociat:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="asob3Box">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>430</y>
     <width>501</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="salveaza_2">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>470</y>
     <width>190</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Salveaza</string>
   </property>
  </widget>
  <widget class="QPushButton" name="anuleaza">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>510</y>
     <width>190</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Anuleaza</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
